### ISSUE TITLE
Custom shortcuts for menus and everything inside the editor

### ISSUE THREAD

**Issue by smedelyan** [godotengine/godot#25634]
# Feature proposal
**Godot version:** 3.0.6

**TL;DR: Please, extend the list of available shortcuts in the Editor -> Editor Settings -> Shortcuts so that it contains main window menu items as well**

Hi all,
I'm following the tutorial and suddenly I realizied that I had to navigate to Project -> Project Settings quite often during the last two days. That led me to the idea of learning the shortcut for this menu item. Though, there's no such, I hoped to find something related at Editor -> Editor Settings -> Shortcuts, but there's totally nothing related to menus.

Most IDEs (like ones from JetBrains) offer you an ability to customize shortcuts for almost everything, and there's even a default shortcut for, say, project settings. I find that quite convenient, but failed to find such an ability in Godot.

Maybe I just overlooked something?

**Comment by vlogozzo** [comment]
Hello, I'm new to the project and would like to work on this bug. Can anyone let me know where the default shortcuts are saved to, or what DB I should look into.

**Comment by smedelyan** [comment]
@vlogozzo 
I''ve skimmed issues and find this:
https://github.com/godotengine/godot/pull/24486#issuecomment-448923528
https://github.com/godotengine/godot/issues/22883#issuecomment-447785817

https://github.com/godotengine/godot/blob/325efb60b7b4f7cb4ed11097c8f2e2de973e00d0/editor/editor_settings.h#L101

I hope that would be enough to start with=)

**Comment by walstap** [comment]
Hi vlogoozzo,

I am also new and working on this problem. 
Have a look at editor/editor_settings.cpp, lines 1388 until 1475 for function defintions.  
For examples shortcut settings look at editor/editor_nodes.cpp line 5257. 
Maybe we can cooperate and split some tasks.

**Comment by vlogozzo** [comment]
@walstap 
I have, I dont think it will be too much work, besides changing the initial script to create an extra key binding on the menu list, everything else should work with the change. Unless you have any ideas of anything else needing to be changed, I don't think there's much work to split up.

**Comment by vlogozzo** [comment]
@walstap I've stop working on this if your interested

**Comment by YasmineBOU** [comment]
Hi!

We're students of the University of Paris 8 in France.
In one of our subjects, we have to contribute in an open source project.

We would like to know if you're always working on this bug because we would like to work on it?

**Comment by akien-mga** [comment]
Fixed by #26551.
